ui/pages/permissions-connect/permissions-connect.test.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import { ApprovalType } from '@metamask/controller-utils';
+import { BtcAccountType } from '@metamask/keyring-api';
+import messages from '../../../app/_locales/en/messages.json';
+import { renderWithProvider } from '../../../test/lib/render-helpers';
+import mockState from '../../../test/data/mock-state.json';
+import { CONNECT_ROUTE } from '../../helpers/constants/routes';
+import { createMockInternalAccount } from '../../../test/jest/mocks';
+import { shortenAddress } from '../../helpers/utils/util';
+import PermissionApprovalContainer from './permissions-connect.container';
+
+const mockPermissionRequestId = '0cbc1f26-8772-4512-8ad7-f547d6e8b72c';
+
+jest.mock('../../store/actions', () => {
+  return {
+    getRequestAccountTabIds: jest.fn().mockReturnValue({
+      type: 'SET_REQUEST_ACCOUNT_TABS',
+      payload: {},
+    }),
+  };
+});
+
+const mockAccount = createMockInternalAccount({ name: 'Account 1' });
+const mockBtcAccount = createMockInternalAccount({
+  name: 'BTC Account',
+  address: 'bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq',
+  type: BtcAccountType.P2wpkh,
+});
+
+const defaultProps = {
+  history: {
+    location: {
+      pathname: `${CONNECT_ROUTE}/${mockPermissionRequestId}`,
+    },
+  },
+  location: {
+    pathname: `${CONNECT_ROUTE}/${mockPermissionRequestId}`,
+  },
+  match: {
+    params: {
+      id: mockPermissionRequestId,
+    },
+  },
+};
+
+const render = (
+  props = defaultProps,
+  type: ApprovalType = ApprovalType.WalletRequestPermissions,
+) => {
+  let pendingPermission;
+  if (type === ApprovalType.WalletRequestPermissions) {
+    pendingPermission = {
+      id: mockPermissionRequestId,
+      origin: 'https://.metamask.io',
```

Comment:
IDK if the `.` was intended?

```suggestion
      origin: 'https://metamask.io',
```